.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Can we add `.idea` folder to .gitignore?

Reply:
My mistake, that would be fine!